hack/make-rules/test-cmd-util.sh

```diff
@@ -915,6 +915,32 @@ __EOF__
   set +o errexit
 }
 
+# runs specific kubectl create tests
+run_create_tests() {
```

Comment:
since you create a new func here, could we make the name more specific, like run_dryrun_override_namespace or sth better ?

Reply:
@shiywang Thanks for the feedback, since this has already merged, will go ahead and sneak in a better name for this function as part of this PR https://github.com/kubernetes/kubernetes/pull/54083 which updates test-cmd-util.sh anyway :)